rosdep/python.yaml

```diff
@@ -1573,6 +1573,13 @@ python-genshi:
   fedora: [python-genshi]
   gentoo: [dev-python/genshi]
   ubuntu: [python-genshi]
+python-geoalchemy2-pip:
+  debian:
+    pip:
+      packages: [geoalchemy2]
+  ubuntu:
```

Comment:
If we're doing a `-pip` package, we may as well add support for fedora and osx as well (like https://github.com/ros/rosdistro/blob/master/rosdep/python.yaml#L157).  Thanks.